### ISSUE TITLE
Maven Central release

### ISSUE THREAD

**Issue by ginkel** [esl-client/esl-client#22]
Are there any plans for a release on Maven Central? IMHO, that would make using this fork much easier.

Possible issues:

* Which group ID should be used (is there any contact to the original authors so the group id could be transferred)?

**Comment by JTCozart** [comment]
I would like to see a maven central deploy as well.

**Comment by mgodave** [comment]
I'm not actively maintaining this project anymore but if you're using it I'd be happy to add you to the organization and you can figure out how you and the rest of the community would like to manage this.

**Comment by borourke** [comment]
@mgodave yes it would be great if you could add me and @JTCozart to this project/organization as we are still using this ESL client at our company.

**Comment by livem** [comment]
@ginkel @JTCozart @borourke 
That was a good idea , thanks for your attention and suggestions. 
Later I will take a look at how to upload this project to maven central.

By the way , is there easy way github project to maven central ?

**Comment by jamesjieye** [comment]
There is one in maven central, but it was published in 2010.
https://mvnrepository.com/artifact/org.freeswitch.esl.client/org.freeswitch.esl.client/0.9.2

**Comment by jamesjieye** [comment]
The version 0.9.2 in maven central is out dated. It does not contain the latest code in this repository.

**Comment by livem** [comment]
2010


On 4/19/2019 07:55，James J. Ye<notifications@github.com> wrote：

The version 0.9.2 in maven central is out dated. It does not contain the latest code in this repository.

**Comment by livem** [comment]
顺便请教一下，有比较方便的工具可以直接打包 github project to maven central 的吗


On 4/19/2019 07:55，James J. Ye<notifications@github.com> wrote：

The version 0.9.2 in maven central is out dated. It does not contain the latest code in this repository.
